xml/containers-get.xml

```diff
@@ -1,562 +1,635 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE chapter
-[
+<!DOCTYPE chapter [
   <!ENTITY % entities SYSTEM "entity-decl.ent">
-    %entities;
+  %entities;
 ]>
-<chapter version="5.0" xml:id="cha-containers-get"
-  xmlns="http://docbook.org/ns/docbook"
-  xmlns:xi="http://www.w3.org/2001/XInclude"
-  xmlns:xlink="http://www.w3.org/1999/xlink">
- <info>
-  <title>Obtaining Containers</title>
- </info>
- <para>
+<chapter version="5.0" xml:id="cha-containers-get" xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <info>
+    <title>Obtaining Containers</title>
+  </info>
+  <para>
   This chapter provides information on obtaining container images.
  </para>
- <sect1 xml:id="id-sle-base-images">
-  <title>&sle; Base Images</title>
+  <sect1 xml:id="id-sle-base-images">
+    <title>&sle; Base Images</title>
 
-  <para>
+    <para>
    &suse; offers a large number of official base container images that can be
    used as a starting point for building custom containers. Each &slea; base
    image contains a &slea; release without the kernel. The OS included in the
    base image is an empty environment without any functionality.
   </para>
 
-  <para>
-   Base images are available from <link xlink:href="https://registry.suse.com" />. For
-   information about the &suse; Registry, see <xref linkend="suse-registry"/>.
+    <para>
+      Base images are available from
+      <link xlink:href="https://registry.suse.com" />
+      . For
+   information about the &suse; Registry, see
+      <xref linkend="suse-registry" />
+      .
    The images in the &suse; Registry are all Generally Availability (Production)
```

Comment:
```suggestion
   The images in the &suse; Registry are all General Availability (Production)
```

Reply:
I changed this to "Generally Available" which at least grammatically seemed to work better. What you think?

Reply:
I disagree. ISTM that "General Availability" is the official description of SUSE's product release status, so I think we should not adjust it to fit with the rest of the sentence.

And _definitely_ not "Generally Availability"!

Reply:
Right - fixed that then.